### PR TITLE
States explained for Readme.md

### PR DESCRIPTION
**Pull request by RvMp**
Recently explained on Discord and it helped me alot. Could be useful to add this to the README.md. Did some small adjustments. Credits: Spencer Truman

### CODE REVIEW

**Comment by RvMp** [comment]
I don't believe changing the Readme causes a compile error right?

**Comment by cdecker** [comment]
Nope, but flaky tests do :wink: Kicking the CI

File: README.md [hunk 1]
```diff
@@ -104,6 +104,19 @@ This opens a connection and, on top of that connection, then opens a channel.
 The funding transaction needs 1 confirmations in order for the channel to be usable, and 6 to be broadcast for others to use.
 You can check the status of the channel using `cli/lightning-cli listpeers`, which after 1 confirmation should say that `state` is `CHANNELD_NORMAL`; after 6 confirmations you can use `cli/lightning-cli listchannels` to verify that the `public` field is now `true`.
 
+### Different states 
+
+States starting with `ONCHAIND` mean that the channel has been closed and an onchain transaction exists reflecting the resulting balances
+* `ONCHAIND_OUR_UNILATERAL` > Closed by you without cooperation of the counterparty
+* `ONCHAIND_THEIR_UNILATERAL` > Closed by the counterparty without your cooperation
+* `ONCHAIND_MUTUAL` > Negotiated closing by both sides(edited)
```
Comment (windsok) [inline README.md]: what do you mean by "(edited)" ?

**Comment by windsok** [comment]
I have one channel stuck in CLOSINGD_SIGEXCHANGE state, would love to get some documentation on that one too.

**Comment by RvMp** [comment]
Solved. Copy-Paste error. "edited" was not meant to be there

**Comment by cdecker** [comment]
Why are you closing the PR? I thought it was looking good, and I was looking into how to generate a `dot` graph to show transitions as well. If you change your mind, this documentation would be very welcome :+1:

**Comment by RvMp** [comment]
Some trouble with my fork. I accidentally changed things to master which prevented me from making some other changes. Will make a new one from a proper branch.

**Pull request closed without merging**